### DISCUSSION TITLE
Welcome to CoreProject-V3-UI Discussions!

### DISCUSSION THREAD

**Discussion by stabldev** [stabldev/coreproject-v3-ui#253] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by baseplate-admin** [comment]
Uhh, I want fish. Do you guys share fish here?

**Reply by stabldev** [reply]
> I want fish. Do you guys share fish here?

LOL
Why not, this is no: 1 fish market
which type you want? blonde? stockings?

**Reply by baseplate-admin** [reply]
>  which type you want? blonde? stockings?

Bothhhhh..

**Comment by baseplate-admin** [comment]
I want melonsssssss

**Reply by stabldev** [reply]
Sorry @boilerplate-admin
We only provide fishes, no fruits ...

**Reply by baseplate-admin** [reply]
i am unsubbing form this :( 

Good luck to your project
